lib/ladybug/energy_model/face.rb

```diff
@@ -0,0 +1,149 @@
+        openstudio_vertices = OpenStudio::Point3dVector.new
+        @hash[:geometry][:boundary].each do |vertex|
+          openstudio_vertices << OpenStudio::Point3d.new(vertex[0], vertex[1], vertex[2])
+        end
+
+        openstudio_construction = nil
+        if @hash[:properties][:energy][:construction]
+          construction_name = @hash[:properties][:energy][:construction]
+          construction = openstudio_model.getConstructionByName(construction_name)
+          unless construction.empty?
+            openstudio_construction = construction.get
+          end
+        end
+
+        openstudio_surface = OpenStudio::Model::Surface.new(openstudio_vertices, openstudio_model)
+        openstudio_surface.setName(@hash[:name])
+        openstudio_surface.setSurfaceType(@hash[:face_type])
+        openstudio_surface.setConstruction(openstudio_construction) if openstudio_construction
+        openstudio_surface.setOutsideBoundaryCondition(@hash[:boundary_condition][:type])
+        boundary_condition = (@hash[:boundary_condition][:type])
+       
+
+        case boundary_condition
+        when 'Outdoors'
+          if @hash[:boundary_condition][:sun_exposure] == true
+            openstudio_surface.setSunExposure('SunExposed')
+          else 
+            openstudio_surface.setSunExposure('NoSun')
+          end
+          if @hash[:boundary_condition][:wind_exposure] == true
+            openstudio_surface.setWindExposure('WindExposed')
+          else
+            openstudio_surface.setWindExposure('NoWind')
+          end
+          if @hash[:boundary_condition][:view_factor] == 'autocalculate'
+            openstudio_surface.autocalculateViewFactortoGround
+          else
+            openstudio_surface.setViewFactortoGround(@hash[:boundary_condition][:view_factor])
+          end
+        when 'Surface'
+          surface = nil 
+          if @hash[:boundary_condition][:boundary_condition_objects][0]
+            surface_object = openstudio_model.getSurfaceByName(@hash[:boundary_condition][:boundary_condition_objects][0])
+            unless surface_object.empty?
+              surface = surface_object.get
+              openstudio_surface.setAdjacentSurface(surface)
+            end
+          end
+        end
+
+
+        if @hash[:apertures]
+          @hash[:apertures].each do |aperture|
+            aperture = Aperture.new(aperture)
+            openstudio_subsurface_aperture = aperture.to_openstudio(openstudio_model)
+            openstudio_subsurface_aperture.setSurface(openstudio_surface)
+          end
+        end
+
+        if @hash[:doors]
+          @hash[:doors].each do |door|
+            door = Door.new(door)
+            openstudio_subsurface_door = door.to_openstudio(openstudio_model)
+            openstudio_subsurface_door.setSurface(openstudio_surface)
+          end
+        end
+
+        if @hash[:indoor_shades]
+          @hash[:indoor_shades].each do |indoor_shade|
```

Comment:
As I said above, indoor shades have no bearing on the final E+ simulation since a `ZoneSurface:Detailed` on the inside of a zone does not influence the calculation at all. So this can be removed here. We only need to worry about `outdoor_shades`.